src/test/sql/sql_pipeline_test.cpp

```diff
@@ -423,14 +428,88 @@ TEST_F(SQLPipelineTest, GetResultTableNoOutput) {
   const auto sql = "UPDATE table_a SET a = 1 WHERE a < 150";
   auto sql_pipeline = SQLPipelineBuilder{sql}.create_pipeline();
 
-  const auto& table = sql_pipeline.get_result_table();
+  const auto& [pipeline_status, table] = sql_pipeline.get_result_table();
+  EXPECT_EQ(pipeline_status, SQLPipelineStatus::Success);
   EXPECT_EQ(table, nullptr);
 
   // Check that this doesn't crash. This should return the previous table.
-  const auto& table2 = sql_pipeline.get_result_table();
+  const auto& [pipeline_status2, table2] = sql_pipeline.get_result_table();
+  EXPECT_EQ(pipeline_status2, SQLPipelineStatus::Success);
   EXPECT_EQ(table2, nullptr);
 }
 
```

Comment:
(random line)
what about testing that `<dml statement>; SELECT * FROM ...; <dml_statement>` returns indeed `{nullptr, a_table, nullptr}`?